ta/bonnie/bonnie.c

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
No issue with license terms of the original bonnie? Did you reuse any code from Bonnie, or only the name?